app/views/shared/_number_of_issues.html.slim

```diff
@@ -0,0 +1,7 @@
+=link_to_if(Category.calculate(id, 'opened') != 0, Category.calculate(id, 'opened'), account_admin_category_path(id: id, status: 'opened'), title: 'Відкриті')
```

Comment:
@denys281, calculate - it is method, that calculate number of issues in category by category ID and issue STATUS. But I see how I can improve it - I will define instance method in Category model.